support/reftest/reftest

```diff
@@ -1,22 +1,22 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 """
 
 Prepare reference test data for Exodus-lambda integration test.
 It requires a "data.yml" file specifying the reference test data.
 
 """
 
-import os
-import sys
-import yaml
-import boto3
+import argparse
```

Comment:
@lioramilbaum  small suggestion: could you pls change the import layout?
```
import sys
import yaml
import boto3
import hashlib
import tempfile
import argparse
import requests

from tqdm import tqdm
from datetime import datetime
```
it will look better.

Reply:
isort is producing the current import layout, according to the best practices. Do we want to override those?

Reply:
@lioramilbaum I just checked the tool `isort`, it's pretty useful :+1: